src/app/logged-in-banner/logged-in-banner.component.scss

```diff
@@ -0,0 +1,32 @@
+.flex-title-container {
+  justify-content: center;
+}
+
+.flex-container {
+  width: 100vw;
+  background: url('../../assets/images/dockstore/ship.png') no-repeat, url('../../assets/images/dockstore/truck.png') no-repeat, #21335b;
+  background-position: 45% 100%, 70% 100%;
+}
+
+.stacked-button {
+  margin: 0.25rem 0rem;
+  font-size: 16px;
+}
+
+.flex-buttons-container {
+  justify-content: center;
```

Comment:
this style is now completely identical to the other above.  furthermore, it may be replaced by fxLayoutAlign="center center"

Reply:
replaced it